Improved help file

zds-dev opened the pull request:
Converted the confusing 'time' in the help documentation into seconds so it is more understandable. Possibly could be converted into minutes or something. But I left it at seconds.

zds-dev committed:
Update help_market.js

Converting milliseconds into seconds?

zds-dev committed:
Update help_market.html

Changed income interval into seconds

zds-dev committed:
Merge pull request #1 from SolitudalDeveloper/patch-2

Update help_market.html

dan335 commented on packages/dominus-help/client/help_market.html:
```diff
@@ -19,7 +19,7 @@
 Villages collect resources from the 6 hexes surrounding them.
 
-Resources are collected every {{incomeInterval}}.
+Resources are collected every {{round incomeInterval}} seconds.
```
Should remove round and change seconds to minutes.  Rounding should be done in the .js file.

dan335 commented on packages/dominus-help/client/help_market.js:
```diff
@@ -12,6 +12,6 @@ Template.help_market.helpers({
 
   incomeInterval: function() {
-    return _gs.castles(Session.get('gameId'), 'incomeInterval');
+    return _gs.castles(Session.get('gameId'), 'incomeInterval') / 60 ; /* ms to seconds */
```
Should change to 
```
return Math.round(_gs.castles(Session.get('gameId'), 'incomeInterval') / 1000 / 60);
```
There are 1000 ms in a second.  Also minutes would be better I think.

zds-dev replied:
Indeed. I'm not sure how I managed to get my maths off. I'll sort that.

zds-dev committed:
Update help_market.js

Income interval converted into minutes from ms

zds-dev committed:
Update help_market.html

Removed round since it was added in .js.
Changed 'seconds' to 'minutes' since the .js now returns minutes.

dan335 commented:
merged!  thanks
